lib/checks/label/duplicate-img-label-evaluate.js

```diff
@@ -1,23 +1,21 @@
 import { getRole } from '../../commons/aria';
 import { visibleVirtual, accessibleTextVirtual } from '../../commons/text';
-import { findUpVirtual } from '../../commons/dom';
-import { getNodeFromTree } from '../../core/utils';
+import { closest } from '../../core/utils';
 
-function duplicateImgLabelEvaluate(node, options, virtualNode) {
+function duplicateImgLabelEvaluate(node, options = {}, virtualNode) {
 	if (['none', 'presentation'].includes(getRole(node))) {
 		return false;
 	}
 
-	const parent = findUpVirtual(
-		virtualNode,
-		'button, [role="button"], a[href], p, li, td, th'
-	);
+	const {
+		parentSelector = 'button, [role="button"], a[href], p, li, td, th'
```

Comment:
I think we should move all the defaults into the options object in the JSON, so that it is always in the same place and not some of it in options and some of it in evaluate.